pkgs/tools/networking/network-manager/applet.nix

```diff
@@ -20,8 +20,18 @@ in stdenv.mkDerivation rec {
       url = https://gitlab.gnome.org/GNOME/network-manager-applet/merge_requests/12.patch;
       sha256 = "0q5qbjpbrfvhqsprnwjwz4c42nly59cgnbn41w2zlxvqf29gjvwk";
     })
+    (fetchpatch {
+      # https://gitlab.gnome.org/GNOME/network-manager-applet/issues/11
+      # should be fixed in 1.8.15
```

Comment:
1.8.15 is a dev version (unlike rest of GNOME, NM stable versions need to have even patches in addition to even minors), next release will be 1.8.16.

Reply:
didn't knew that, ok - updated string to 1.8.16.